pkg/util/orchestrator/fake/client.go

```diff
@@ -65,6 +65,38 @@ func (o *FakeOrc) AddInstance(cluster, host string, master bool, sls int64, slav
 	o.Clusters[cluster] = []Instance{inst}
 }
 
+func (o *FakeOrc) AddInstanceInTopology(cluster, host string, port int, master bool, sls int64, slaveR, upToDate bool, masterKey string, masterPort int, coMaster bool) {
```

Comment:
I propose that, lets return a pointer to newly created `Instance` so you can easily modify instance so you don't need to pass that much parameters to this function.

And also you can use `AddInstance` function from above.

```go
func (o *FakeOrc) AddInstance(cluster, host string, master bool, sls int64, slaveR, upToDate bool) *Instance
```
By keeping the old parameters you don't have to modify old tests. The new test will lock like:
```go
inst := orcClient.AddInstanceInTopology("asd.default", "foo122-mysql-0",
					3306, false, -1, false, true)
inst.MasterKey = InstanceKey{Hostname: "foo122-mysql-5", Port: 3305}
inst.IsCoMaster = true
```